### DISCUSSION TITLE
Step increments (H2)

### DISCUSSION THREAD

**Discussion by SteveBon** [kachurovskiy/nanoels#97] [Q&A]
After having a play with the controller in my front room this evening. Am I correct that the steps button (S) should scroll through 1.000, 0.100, 0.010 & then 0.001? Mine reads  "blank", 0.100, 0.010 and then 0.001
However, when I press + or - the maximum increments I get are 0.010mm. The "blank" (should be 1.000mm?) and 0.100 only increment/decrement by 0.010mm per press. 0.010 increments/decrements by 0.010mm, the 0.001 increments/decrements by 0.001
Many thanks again

**Comment by kachurovskiy** [comment]
Hi @SteveBon, see https://github.com/kachurovskiy/nanoels/tree/main/h2#moving-the-carriage for the left/right moving logic. As you see, it depends on whether ELS is ON/off and your pitch setting when it's ON. It would help if you could take a video of your problem and attach it here. Many thanks.

**Reply by SteveBon** [reply]
I've been playing again tonight after re-reading your instructions. Thank you, I now understand.
Do you know if anyone has modeled a 58 tooth encoder wheel like your conical one? These "clone" machines all vary. Mine's 58 tooth.

**Reply by SteveBon** [reply]
@kachurovskiy I've just modified your encoder gear and now printing it.
I'll keep you posted 😊

**Reply by SteveBon** [reply]
Well, it turns out I didn't read the label properly. Mine is a 56 tooth spindle. All up and running nicely now.
Thanks again Maxim, excellent work from you. I've done a brief write up on a forum I frequent - hopefully your YT channel will get some more hits soon!

**Reply by kachurovskiy** [reply]
Awesome, congratulations! Please post a few pictures of your build on https://github.com/kachurovskiy/nanoels/discussions if you get a chance :) Thanks!
